### DISCUSSION TITLE
[v3.2.0--alpha.xx] Laden der ersten Seite verhindern

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2539] [General]
Kann man das Laden der ersten Seite verhindern, wenn man eine andere im hash ausgewählt hat?
Evtl display der Seite erst dann einschalten, wenn gewünschte Seite vorhanden?

**Comment by Zefau** [comment]
Hatte schon mitbekommen, dass scheinbar bei einigen Usern zunächst der erste Tab lädt und jarvis dann wechselt. Bei mir läd direkt der korrekte Tab ohne vorher bei Index 0 zu starten.

Du rufst quasi `http://<ip>:<port>/jarvis/#/c9ae69c1-5c33-49bb-86e9-6ca2106cb9b7` auf, richtig?

**Reply by mcuiobroker** [reply]
Ja

**Reply by Zefau** [reply]
Habe mal ein Bug draus gemacht, aber bin mir unschlüsslig, wie ich das nachgestellt bekomme.

**Comment by Zefau** [comment]
https://github.com/Zefau/ioBroker.jarvis/issues/2545
